src/api/veilarbperson.ts

```diff
@@ -54,6 +54,31 @@ export interface VergeOgFullmakt {
     fullmakt: Fullmakt[];
 }
 
+export interface RepresentasjonFullmakt {
+    fullmakt: FullmaktData[];
+}
+
+export interface FullmaktData {
+    fullmaktsgiver: StringOrNothing;
+    fullmektig: StringOrNothing;
+    omraade: OmraadeMedHandling[];
+    gyldigFraOgMed: StringOrNothing;
+    gyldigTilOgMed: StringOrNothing;
+    fullmaktsgiverNavn: StringOrNothing;
+    fullmektigsNavn: StringOrNothing;
+}
+
+export interface OmraadeMedHandling {
+    tema: StringOrNothing;
+    handling: OmraadeHandlingType[];
+}
+
+export enum OmraadeHandlingType {
+    LES = 'LES',
+    KOMMUNISER = 'KOMMUNISER',
+    SKRIV = 'SKRIV'
+}
```

Comment:
I og med at vi ikke bruker denne dataen til noe, kunne vi unngått å lagre den? Sånn for dataminimeringens skyld, kan vi forenkle modellen til å bare være om det eksisterer en fullmakt eller ikke? 🤔

Reply:
Det er egentlig god ide og jeg selv hadde ønske å gjøre det sånn mange år siden når jeg jobbet med visistkortet men Alexander ville ikke ha det sånn. Jeg tenker hvis vi skal lagre bare statusen om respons inneholder data da er det nok for å vise etiketten fordi vi bruker ikke denne data til noe. Jeg tenker vi kan gjøre det likt for alle de data henterne som blir brukt bare for å vise etiketten men vi bruker ikke den dataen til noe.

Reply:
Ja! Det er noe vi kunne sett på. Hadde jo vært fint for å redusere størrelsen på appen litt :)